src/display/Makefile.am

```diff
@@ -5,7 +5,7 @@ AUTOMAKE_OPTIONS = 1.6.3
 noinst_LIBRARIES = libdisplay.a
 
 libdisplay_a_SOURCES = display.c display.h display_stdio.c display_stdio.h \
-	textblock.c textblock.h \
+	textblock.c textblock.h unicode.c unicode.h\
```

Comment:
I believe we only want to include this in the SDL build. That should look something like:

1. Move `unicode.c unicode.h` into `OPTIONAL_SRC` below
2. Add it to `display_objects` in _configure.ac_ if SDL is enabled.